server/src/main/java/org/opensearch/gateway/remote/RemoteClusterStateService.java

```diff
@@ -88,6 +91,8 @@ public class RemoteClusterStateService implements Closeable {
     public static final TimeValue GLOBAL_METADATA_UPLOAD_TIMEOUT_DEFAULT = TimeValue.timeValueMillis(20000);
 
     public static final TimeValue METADATA_MANIFEST_UPLOAD_TIMEOUT_DEFAULT = TimeValue.timeValueMillis(20000);
+    public static final TimeValue CLUSTER_STATE_CLEANUP_INTERVAL_DEFAULT = TimeValue.timeValueMillis(60000);
+    public static final TimeValue CLUSTER_STATE_CLEANUP_INTERVAL_MINIMUM = TimeValue.MINUS_ONE;
```

Comment:
If we set interval to -1, the task will not get scheduled and cleanup will be paused. Have kept this setting, so that we have an option to pause clean up if required in any case.